ush/python/pygfs/task/marine_letkf.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+from logging import getLogger
+from pygfs.task.analysis import Analysis
+from typing import Dict
+from wxflow import (chdir,
+                    logit,
+                    Task)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class MarineLETKF(Analysis):
+    """
+    Class for global ocean analysis LETKF task
```

Comment:
Is this ocean or marine? it might not be obvious to all what the distinctions are.

Reply:
Good point. How about keeping marine in the class name but precising that this implementation in only for  the ocean and seaice?

Reply:
@guillaumevernieres Any preference?